.github/workflows/tag-release.yml

```
@@ -0,0 +1,92 @@
+    permissions:
+      contents: write
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout 🛎️
+        uses: actions/checkout@de0fac2e4500dabe0009e67214ff5f5447ce83dd # v6
+        with:
+          fetch-tags: false
+
+      - name: Setup workspace ⚙️
+        uses: ./.github/actions/setup-workspace
```

actionlint (1.7.12)

[error] description is required in metadata of "Setup workspace" action at "/home/jailuser/git/.github/actions/setup-workspace/action.yml"

(action)